scripts/set_empty_upload_args.sh

```diff
@@ -7,3 +7,8 @@ codecov_args+=( $(write_bool_args CODECOV_FORCE) )
 codecov_args+=( $(k_arg GIT_SERVICE) $(v_arg GIT_SERVICE))
 codecov_args+=( $(k_arg SHA) $(v_arg SHA))
 codecov_args+=( $(k_arg SLUG) $(v_arg SLUG))
+
+# Set the arguments to create commit
+codecov_args+=( $(k_arg PARENT_SHA) $(v_arg PARENT_SHA))
```

Comment:
I tried to keep these in roughly alphabetic order for sanity in the future. I don't think we'll need the comment if it's documented on the CLi side